src/app/services/articles-service/index.ts

```diff
@@ -0,0 +1,46 @@
+import dayjs from 'dayjs';
+// eslint-disable-next-line
+// @ts-ignore
+import { API_KEY as apiKey } from 'react-native-dotenv';
+import ApiService from '@services/config';
+import {
+  topicValues,
+  sortValues,
+  timeValues,
+} from '@modules/MainArticlesScreen/namespace/index';
+
+const now = dayjs();
```

Comment:
Should now be here? It get now() when js is loaded for the first time at the beggining, not when specific service is called. So if you open application and leave it for hour or 2, and call the servioce the now will point to the time before current time.